lib/vector-core/src/stream/driver.rs

```diff
@@ -168,7 +168,8 @@ where
     ) {
         match result {
             Err(error) => {
-                emit(service::CallError { error, request_id });
+                // `Error` and `EventsDropped` internal events are emitted in the sink retry logic.
+                error!(message = "Service call failed.", ?error, request_id);
```

Comment:
Do we need this log at all then? Maybe it could be a `debug` level log?

Reply:
Good question.
The only concern I have with making it debug (and I'm not sure the answer) , is would that make the `request_id` less accessible? I am not sure how important that would be in a incident in the field etc. It _seems_ more like a debug level info :thinking:

Reply:
I can change the log level in my other PR , if we decide to go with debug, since I'll be merging this in there.

Reply:
👍 we can discuss in the other PR. I guess I figured whatever event was being duplicated here would also have the request id.